index.pug

```diff
@@ -0,0 +1,581 @@
+          img.lazy.boxicon(data-src="/assets/education/multiqr.svg", alt="multiqr")
+          h2 MultiQR
+          ul
+            li Accept UPI payments for FREE
+            li Personalised QR with your business logo
+            li Generate unlimited QRs. No restrictions.
+            li SMS and email alerts for all payments
+          span Explore &gt;
+      .sec7box1
+        a(href='https://razorpay.com/payments-app/' target='_blank')
+          svg.svg-corner.svg-desktop(xmlns='http://www.w3.org/2000/svg' width='316' height='350' fill='#0f1b38') path.border(d='M.5 219.511V6A5.5 5.5 0 016 .5h216.883c4.105 0 8.042 1.628 10.948 4.527l77.116 76.937a15.499 15.499 0 014.553 10.973v190.054a5.501 5.501 0 01-5.509 5.5l-304-.48a5.5 5.5 0 01-5.491-5.5z' stroke='#4aeccb')
+          img.lazy.boxicon(data-src="/assets/education/mobileapp.svg", alt="mobile-app")
+          h2 Mobile App
+          ul
+            li Accept payments at store and on the go
+            li Collect in-store card payments
+            li Get detailed payment insights
+            li Backed by the safety &amp; security of Razorpay
+          span Explore &gt;
+      .sec7box1
+        a(href='https://razorpay.typeform.com/to/FuLkWeaT#email=xxxxx' target='_blank')
+          svg.svg-corner.svg-desktop(xmlns='http://www.w3.org/2000/svg' width='316' height='350' fill='#0f1b38') path.border(d='M.5 219.511V6A5.5 5.5 0 016 .5h216.883c4.105 0 8.042 1.628 10.948 4.527l77.116 76.937a15.499 15.499 0 014.553 10.973v190.054a5.501 5.501 0 01-5.509 5.5l-304-.48a5.5 5.5 0 01-5.491-5.5z' stroke='#4aeccb')
+          img.lazy.boxicon(data-src="/assets/education/taponglass.svg", alt="taponglass")
+          h2 Tap on Glass
+          ul
+            li Collect card payments using your NFC android smartphone
+            li Let customers tap their credit &amp; debit cards to pay
+          span Sign up for early access &gt;
+#MobileAccordian
+  .accordion_container
+    .accordion_head
+      | Start online business 
+      span.plusminus +
+    .accordion_body
+      section#StartOnlineBusiness.container1.section4
+        img.lazy.boxicon(data-src="/assets/education/sec4bg.svg", alt="sec4back-shape")
+        h1 Start selling online without a website
+        p
+          | Set up your business and accept payments for startups, entrepreneurs and freelancers who are just starting their business journey
+        ul.sec4pointers
+          li No coding required
+          li Easy to use
+          li Go live in few minutes
+        .sec4inside
+          .sec4box1
+            a(href='https://razorpay.typeform.com/to/e7htPvlI#email=xxxxx' target='_blank')
+              svg.svg-corner.svg-desktop(xmlns='http://www.w3.org/2000/svg' width='316' height='350' fill='#0b1b42') path.border(d='M.5 219.511V6A5.5 5.5 0 016 .5h216.883c4.105 0 8.042 1.628 10.948 4.527l77.116 76.937a15.499 15.499 0 014.553 10.973v190.054a5.501 5.501 0 01-5.509 5.5l-304-.48a5.5 5.5 0 01-5.491-5.5z' stroke='#4aeccb')
```

Comment:
why SVG's are included in the HTML? please move them to a different file and import here. 
all the SVG's